mcda/electre/class_assignment.py

```diff
@@ -0,0 +1,194 @@
+from typing import Any, Dict, Union
+
+from ..core.aliases import NumericValue
+from ..electre.outranking import OutrankingRelation
+
+import pandas as pd
+
+
+def get_relation_type(
```

Comment:
We have `outranking_relation_marginal()` in the outranking.py module (and as I can see it does the same job). Cannot we just use it?